### ISSUE TITLE
[BUG] Killswitch makes the cli unable to connect to the VPN.

### ISSUE THREAD

**Issue by Phate6660** [ProtonVPN/protonvpn-cli-ng#16]
**Describe the bug**
When enabling the killswitch, the cli errors out and I have no internet access until I disconnect from the VPN. The VPN works perfectly fine when the killswitch is disabled, and it only errors out *after* enabling the killswitch.

**To Reproduce**
1. Enable the killswitch with the configure option.
2. Attempt to connect to the VPN.
3. Receive error.
4. Lose internet access.

**Expected behavior**
I expected to be able to connect to the VPN.

**Error Messages/Program Output/Log Messages (~/.pvpn-cli/pvpn-cli.log)**
```
Connecting to US-IL#5 via UDP...
[!] There was an error connecting to the ProtonMail API.
[!] Please make sure your connection is working properly!
```

**Desktop (please complete the following information):**
 - OS: Gentoo
 - Python Version: 3.6.9
 - ProtonVPN-CLI Version: 2.1.0

**Comment by Rafficer** [comment]
So just to get this correctly:

You enabled the Kill Switch without being connected and your internet working, then tried to connect and immediately got this error?

Can you please attach the following things, all while it's in this disconnected state: 

* output of `sudo iptables -S`
* output of `tail -n 30 ~/.pvpn-cli/pvpn-cli.log`

**Comment by Phate6660** [comment]
> You enabled the Kill Switch without being connected and your internet working, then tried to connect and immediately got this error?

Yep.

> Can you please attach the following things, all while it's in this disconnected state:

`$ sudo iptables -S`
```
-P INPUT ACCEPT
-P FORWARD ACCEPT
-P OUTPUT ACCEPT
-A FORWARD -m state --state RELATED,ESTABLISHED -j ACCEPT
-A FORWARD -i br0 -o eno1 -j ACCEPT
```

`$ tail -n 30 ~/.pvpn-cli/pvpn-cli.log`
```
 'c': False,
 'configure': False,
 'connect': False,
 'd': True,
 'disconnect': False,
 'examples': False,
 'init': False,
 'r': False,
 'reconnect': False,
 'refresh': False,
 's': False,
 'status': False}
2019-11-16 11:02:25,363 — protonvpn-cli — DEBUG — cli:89 — USER: valley
2019-11-16 11:02:25,363 — protonvpn-cli — DEBUG — cli:90 — CONFIG_DIR: /home/valley/.pvpn-cli
2019-11-16 11:02:25,375 — protonvpn-cli — DEBUG — disconnect:304 — Initiating disconnect
2019-11-16 11:02:25,381 — protonvpn-cli — DEBUG — is_connected:178 — Checking connection Status. OpenVPN processes: 1
2019-11-16 11:02:25,896 — protonvpn-cli — DEBUG — is_connected:178 — Checking connection Status. OpenVPN processes: 1
2019-11-16 11:02:26,112 — protonvpn-cli — DEBUG — is_connected:178 — Checking connection Status. OpenVPN processes: 0
2019-11-16 11:02:26,118 — protonvpn-cli — DEBUG — is_connected:178 — Checking connection Status. OpenVPN processes: 0
2019-11-16 11:02:26,119 — protonvpn-cli — DEBUG — manage_dns:608 — Restoring DNS
2019-11-16 11:02:26,121 — protonvpn-cli — DEBUG — manage_dns:620 — resolv.conf changed. Not restoring.
2019-11-16 11:02:26,121 — protonvpn-cli — DEBUG — manage_dns:623 — resolv.conf.backup removed
2019-11-16 11:02:26,122 — protonvpn-cli — DEBUG — manage_ipv6:690 — Restoring IPv6
2019-11-16 11:02:26,133 — protonvpn-cli — DEBUG — manage_ipv6:740 — Removing IPv6 backup file
2019-11-16 11:02:26,134 — protonvpn-cli — DEBUG — manage_ipv6:742 — IPv6 restored
2019-11-16 11:02:26,134 — protonvpn-cli — DEBUG — manage_killswitch:761 — Restoring iptables
2019-11-16 11:02:26,134 — protonvpn-cli — DEBUG — manage_killswitch:763 — Restoring IPTables rules
2019-11-16 11:02:26,139 — protonvpn-cli — DEBUG — manage_killswitch:766 — iptables restored
2019-11-16 11:02:26,139 — protonvpn-cli — DEBUG — manage_killswitch:768 — iptables.backup removed
2019-11-16 11:02:26,139 — protonvpn-cli — DEBUG — disconnect:335 — Disconnected
```

**Comment by Rafficer** [comment]
The debug log is from a disconnection. Are you sure you sent me the logs while your connection wasn't working as you described in the issue? The iptables output also looks like the Kill Switch was active, it wouldn't have accept policies then.

**Comment by Phate6660** [comment]
Oh. I'm sorry. I misinterpreted what you said. I'll attach the correct output.

----

Well this is a bit odd... it's working now. Oh I'm an idiot. I think I realize the problem now.
I was having trouble with `dhcpcd` overwriting my `/etc/resolv.conf` and I just fixed it a couple minutes ago. I must've been having DNS issues while attempting to connect which lead to the errors. I'm sorry for wasting your time with this issue. I'll close this now.
